Jessy/email_ids_challenge.py

```diff
@@ -31,22 +32,33 @@
 names = ['Max Bartlett', 'Angelita Norris', 'Stewart Mueller', 'Dominique Henry', 'Carmela Gross', 'Bettie Mcmillan', 'Sara Ellison', 'Ira Anthony', 'Pauline Riley', 'Ben Weber',
          'Joanne Mcknight', 'Loren Gould', 'Jamar Singh', 'Amanda Vance', 'Tyrell Andrade', 'Jana Clements', 'Eddy Mcbride', 'Marsha Meyer', 'Elbert Shannon', 'Alyce Hull']
 
-emails = []
+
+lower_names=[]
 for i in names:
-    emails.append(i+"@ripplemedia.com")
+    
+    joined_names=''.join(names)
+    # print(i)
+    i.lower()
+    lower_names.append(i.lower())
+
+
+print(joined_names) 
 
-print(emails)
-# We want to convert all their names into the same format from Question 1
-# 2.1 TODO: Use a "for" loop to go over each name in the names list
-for expansion in names:
-    print(expansion)
-# print()
-# 2.2 TODO: Inside the "for" loop, create the email id by re-using the logic from Question 1 and...
+#
+# # print(emails)
+# # We want to convert all their names into the same format from Question 1
+# # 2.1 TODO: Use a "for" loop to go over each name in the names list
+# 
+# # 2.2 TODO: Inside the "for" loop, create the email id by re-using the logic from Question 1 and...
+# for iii in names_list:
 
-for ripplemedia in names:
-    ripplemedia= "@ripplemedia, ".join(names)
+#     email=expansion+'@ripplemedia.com'
+# # for ripplemedia in names:
+# #     ripplemedia= "@ripplemedia, ".join(names)
 
-    print(ripplemedia)
-    break
```

Comment:
A few useful statements (commands) you can use in loops? Break, continue, pass. We can try a few examples of each during a check in or when we have break out sessions soon